databricks/koalas/tests/test_series.py

```diff
@@ -1123,6 +1123,9 @@ def test_astype(self):
         kser = ks.Series(pser)
 
         self.assert_eq(kser.astype(bool), pser.astype(bool))
+        # TODO: restore after pandas 1.1.4 is released.
+        # self.assert_eq(kser.astype(str).tolist(), pser.astype(str).tolist())
+        self.assert_eq(kser.astype(str).tolist(), ["hi", "hi ", " ", " \t", "", "None"])
```

Comment:
After pandas 1.1.4 is released and fixed related bug, I'll come back here to restore this.